tests/run/large_integer_T5290.py

```diff
@@ -0,0 +1,19 @@
+# FIXME: cython: test_assert_c_code_has = __pyx_int_large_498767777577077770_xxx_709777877895072797
+# FIXME: cython: test_assert_c_code_has = __pyx_int_large2_498767777577077770_xxx_709777877895072797
```

Comment:
Maybe it was lost in the context, but this part is the only thing that was failing, and hopefully it's clear now the point I was making that if this test runs successfully than we can assume that there are no collitions

I think that we can just add a comment to the code to update the test in case the amount of digits left will change